src/cirrus/release.py

```diff
@@ -19,7 +19,7 @@
 from cirrus.documentation_utils import (
     build_docs,
     build_doc_artifact,
-    upload_documentation
+    publish_documentation
```

Comment:
makes sense, was kind of thinking that myself as I was going through this again. I'll strip it from the release command